README.md

```diff
@@ -120,9 +131,20 @@ If user wants to manually connect to it then the screen shows the network name s
 For devices having Android version 7.0 and 7.1 user will be sent to settings to enable the wifi hotspot functionality. The user will then need to tap the device back button to come back to skunkworks-crow.
 And for other device it will be enabled automatically.
 
+#### 3.2.2 Bluetooth<a name="send-bluetooth"></a>
+
+We can use bluetooth to send forms as well. The bluetooth has a shorter transferring distance than hotspot, so you may need to put sender and receiver closed to establish a valid connection. In the receiver's settings page, you can enable the secure mode of bluetooth, it allows you to establish an encrypted transmission (needs a pair code when first pairing).
```

Comment:
> We can use bluetooth to send forms as well.

This line can be removed. Since we have already mentioned it above, so no need to mention again